### ISSUE TITLE
Scenario Archive plugin MVP

### ISSUE THREAD

**Issue by leendertvanwolfswinkel** [nens/threedi-results-analysis#937]
Any simulations older than 7 days are no longer available for download from the 3Di API. Customers with a Lizard subscription should be able to work with their 3Di scenarios stored in Lizard seamlessly. Making this a separate plugin improves maintainability.

**Features**
- Search & list available scenario's
- Add raster results as WMS / WMS-T layers (see https://docs.lizard.net/e_lizardwms.html#di-scenarios). 
    - Force QGIS to use `text/plain` as default Identify method? From Lizard dev team: "We added the info_format `text/plain`, which QGIS can work with. You will get this as the "Text" option in the dropdown in the "Identify Results" panel. Unfortunately, QGIS chooses the "Feature" option by default (based on `application/json`), which cannot be read properly. We have not been able to change this, because otherwise we would become backwards incompatible with other existing applications. Perhaps something can be done with the QGIS configuration to change the default, but my QGIS knowledge does not extend far enough for that."
    - Automatically set the identify panel to "Table" after adding this WMS layer [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMVpsQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--abc451b2ba323cfbd34fcb5a61a93d9a32be7af6/image%20%281%29.png)

- Authentication 
    - Logging in should be as easy as possible. Preferably users just provide their username/password in a dialog in QGIS. But probably this is not possible and they will have to generate an API key and copy-paste it into QGIS. If so, to make it as easy as possible for users:
    - Button links to API Key management page https://{organisation_name}.lizard.net/management/personal_api_keys
    - API Key is stored in QGIS password manager

**Existing code & examples**
This python package (developed by 3Di user @emielverstegen) may be helpful: https://github.com/nens/threedi-scenario-downloader. This python API tutorial shows how to download 3Di rasters: https://demo.lizard.net/media/tutorials/How_to_download_a_maximum_waterdepth_raster_from_a_3Di_scenario_stored_in_the_Scenario_Archive_in_Lizard_.ipynb.

**UI sketch**
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdmxYIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b8dd96f72fdf5b71ea3d74d02e503ebc67c2ddfe/image.png)

**Comment by leendertvanwolfswinkel** [comment]
@ldebek Please also add the Model name to the "Scenario" table:
[Name] [Model name] [Organisation] [Created] [UUID]

**Comment by ldebek** [comment]
@leendertvanwolfswinkel MVP version of the plugin is ready for testing. Plugin repository is available here: https://github.com/nens/threedi-scenario-archive

**Comment by leendertvanwolfswinkel** [comment]
@ldebek Can you please rename the repo and module name to `scenario-archive-qgis-plugin`, as we aim to make this a model-agnostic thing, that should also be compatible with results from other hydrodynamic modelling softwares.

**Comment by leendertvanwolfswinkel** [comment]
@ldebek I tested it a bit more and I have very little feedback, other than that I am super happy with this long overdue functionality.

Some minor things before we can release the first version:
- [x] Make the dialog wide enough to show all columns
- [x] Show a feedback message "WMS layers for scenario {scenario name} added to project". Perhaps we can log this to a Logging box at the bottom of the dialog, because I think this kind of user feedback is going to be more relevant when we add more functionality, such as downloading.
- [x] Remove the UI elements related to downloading (unless you think this will be finished really quickly - I saw you already started working on it. I had in mind to release the WMS-only as MVP, than do some other stuff, and than add the download functionality. But if the download stuff is a quick add-on, doing this immediately is also fine for me.)
- [x] With regard to the name, I discussed with Jonas and we came to the conclusion that the name for the plugin should just be "Lizard", and the name for the repo `lizard-qgis-plugin`. I think `lizard_qgis_plugin` would also be a logical name for the module.
